specs/commandline/list.md

```diff
@@ -29,6 +29,7 @@ Aliases:
 Flags:
   -d, --debug             debug mode
   -h, --help              help for list
+      --oci-layout        [Preview] list signatures stored in OCI image layout
```

Comment:
In general, [Preview] means this is a new feature for an early preview and is on track to become a permanent part of the product. It may be turned off only if this feature runs into a problem in the future.

[Experimental] means this new feature can radically change or completely disappear at any time. It's not recommended for production use.

So [Preview] may be still necessary for `--oci-layout` in case this flag is renamed or changed in the future.

Reply:
@FeynmanZhou Thanks Feynman for the explanation. We have a work item in Ga for feature maturity. I think at least one job we need to do is to document the meaning of the labels for feature maturity. cc @toddysm for comments.

Reply:
The definition of [Preview] vs [Experimental] from @FeynmanZhou makes sense to me. In this case I would say the switch `--oci-layout` should be [Experimental].

Reply:
@toddysm as explained here https://github.com/notaryproject/notation/pull/601#discussion_r1155038676, I think `Preview` is more proper. This flag also has the most votes from the community. Both `--target type=oci-layout` (coming in later release) and `--oci-layout` will be supported.